src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 7) {
+    return 230;
```

Comment:
Avoid using hard-coded values like `230` for specific cases. Use constants and calculations to determine the total cost.